lib/vcr.rb

```diff
@@ -186,6 +186,9 @@ def use_cassette(name, options = {}, &block)
 
     begin
       call_block(block, cassette)
+    rescue StandardError
+      cassette.run_failed!
+      raise
```

Comment:
We should raise the exception that gets caught or an exception that wraps the exception caught.

Reply:
`raise` without arguments should re-raise the last exception that was caught (https://ruby-doc.org/core-2.1.2/Kernel.html#method-i-raise). I can agree it would improve readability to explicitly rescue and re-raise the error. Or is there a functional disadvantage to using `raise` with no arguments?

Reply:
:shocked_pikachu: